app/controllers/logs_controller.rb

```diff
@@ -390,6 +380,17 @@ def export
 
   private
 
+  def parse_and_set_start_stop_dates(params)
+    start_date = Date.parse(params['start_date'])
+    stop_date = Date.parse(params['stop_date'])
+
+    params['start_date'] = Hash[month: start_date.month, day: start_date.day, year: start_date.year]
+    params['stop_date'] = Hash[month: stop_date.month, day: stop_date.day, year: stop_date.year]
+
+    @start_date = Date.new(params['start_date']['year'], params['start_date']['month'], params['start_date']['day'])
+    @stop_date = Date.new(params['stop_date']['year'], params['stop_date']['month'], params['stop_date']['day'])
```

Comment:
Can you do `@start_date = Date.new(start_date.year, start_date.month, start_date.day)` here and drop the `Hash` bit?